bodo/utils/transform.py

```diff
@@ -1698,13 +1698,18 @@ def set_ith_arg_to_omitted_value(
     assert call_type.args[pos_idx] == types.Omitted(
         expected_existing_value
     ), f"Omitted({expected_existing_value}) {pos_idx}th argument expected"
-    pass_info.calltypes[rhs] = pass_info.typemap[rhs.func.name].get_call_type(
```

Comment:
I see several more cases of this in other APIs that are perhaps not interacting with streaming. For example `_recompile_func` in distributed analysis. Do these need to be updated as well? If so we should probably encapsulate the return type replacement into a a common API for replacement.

Reply:
I don't think anything else needs to be changed since the streaming change is limited to these cases.